proto/proto/account.proto

```diff
@@ -8,3 +10,9 @@ message AccountId {
     // fixed 64 bits, instead of zig-zag encoding.
     fixed64 id = 1;
 }
+
+message Account {
```

Comment:
Should we name this something like `PrivateAccount` to signify that we don't have the full account data? Note that in `miden-base`, [`Account`](https://github.com/0xPolygonMiden/miden-base/blob/eeef30b05cca4c049e4b7aa4677bf47810701185/objects/src/accounts/mod.rs#L67-L73) is a struct containing all the `Account` data. @bobbinth what do you think?

Reply:
Yeah - probably just `Account` is not ideal here - though I don't love `PrivateAccount` either (as this info can be provided about public accounts too).

Maybe `AccountInfo` would be better.

Reply:
Changed to `AccountInfo`